hpedockerplugin/file_manager.py

```diff
@@ -604,12 +621,39 @@ def mount_share(self, share_name, share, mount_id):
                 }
             }
             share['path_info'] = node_mnt_info
+            if fUser or fGroup or fMode:
+                LOG.info("Inside fUser or fGroup or fMode")
+                is_first_call = True
+                try:
+                    fUName, fGName = self._hpeplugin_driver.usr_check(fUser,
+                                                                      fGroup)
+                    if fUName is None or fGName is None:
+                        msg = ("Either user or group does not exist on 3PAR "
+                               "Please create local users and group with "
+                               "required user id and group is")
```

Comment:
It's better to give information on the actual CLI  command for file persona user/group creations.

Reply:
It is better to refer to 3PAR cli user guide section to create local user and local group on 3PAR.
Hence this information is not given here.